cognee/tests/unit/interfaces/graph/util.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime, timezone
+
+
+def run_test_against_ground_truth(
```

Comment:
I don't like this kind of abstractions because they usually expand over time in order to support more use cases. But let's keep it for now. If it starts growing we can kill it.